### PR TITLE
Add home page

### PR DESCRIPTION
**Pull request by AlduLonghi**
### What does it do?
It creates a replica of [The Next Web](https://thenextweb.com/)

### Tasks 
- [x] Create skeleton HTML
- [x] Add styling 
- [x] Create professional readme file
- [x] Polish the codes and fix linters errors
- [x] Implement all of the requested changes after review

### CODE REVIEW

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🛑

## Hello @AlduLonghi and @descholar-ceo 👋. This is one of the most detailed project I have ever seen 💯 🧨. 

- Awesome README!
- Great use of CSS variables 🚀.
- Great job on avoiding CSS bad practices 👏.

## Just a few issues that are preventing me from approving this project (😩 ).

### Desktop mode

- Increase the size of the TNW logo to match the original.

#### Clone
Annotation 2020-06-25 225624

#### Original
Annotation 2020-06-25 225650


- The `top 3 articles` should be enclosed in `article` tags.

<hr>

### Mobile Screen

- Decrease the height of each of the articles in the `top 3 articles` section to match the original.

#### Clone
image

#### Original
image


- The images in `latest news` section are squeezed.

#### Clone
Annotation 2020-06-25 075128

#### Original
Annotation 2020-06-25 224839

*Possible solution: increase its container width and add `object-fit: cover` property to image.*


- The `Get updates by email` list item is hidden behind the first `div.mid-article-head-content-1`. Make sure that it's visible.

Annotation 2020-06-25 230907


- In `apps, tech, gear,...` section, the height of `div.article-content-2-pic ac2p-1` should be decreased to match the original.

#### Clone
image

#### Original
image


- The images in `latest deals` section are squeezed.

#### Clone
image

#### Original
image

You guys have done an incredible job making this project.

Apply these changes and request another code review.

Happy coding 👨‍💻

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle: 
### HELLO :wave:  @AlduLonghi AWESOME PROJECT SO FAR. AM PETER ROBERT A TSE AND I HAVE REVIEWED YOUR PROJECT :smile: 

**GREAT JOB IMPLEMENTING THE CHANGES BY THE PREVIOUS REVIEW**

your project looks amazing :clap: 

However, there is still an issue with the logo on the desktop view. Kindly you need to fix. 

**YOURS**
twn yours
**ORIGINAL**
twn original

**Kindly fix that and submit the project for another review**

### HAPPY CODING

**Comment by andresporras3423** [comment]
### **Pull Request Under Review By Oscar Russi (TSE)**

File: style.css [hunk 1]
```diff
@@ -0,0 +1,1043 @@
...
+  list-style: disc;
+  color: var(--color-orange);
+  margin-left: 0.2rem;
+  border-bottom: 1px solid var(--color-light-grey);
+  padding: 0.6rem 0;
+}
+
+.latest-deals-grid-header a {
+  color: var(--color-orange);
+}
```
Comment (andresporras3423) [inline style.css]: This selector and selector on line 110 have the same style definition, group selectors to minimize code.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,1043 @@
...
+  border-top: 1px solid rgb(255, 205, 194);
+}
+
+.latest-deals-grid-header a:hover {
+  color: var(--color-black);
+}
+
+.mid-article-head-content-1 a:not(:last-child):hover {
+  color: var(--color-black);
+}
```
Comment (andresporras3423) [inline style.css]: This selector and selectors on lines 94, 132, 464, 505, and 558 all have the same style definition, group selectors to minimize code.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,1043 @@
...
+  background-size: cover;
+}
+
+.ac2p-7 {
+  background: linear-gradient(to bottom, rgba(0, 0, 0, 0.4), rgba(0, 0, 0, 0.4)), url("./assets/article-7.jpg") no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+
+.ac2p-8 {
```
Comment (andresporras3423) [inline style.css]: You are using the same couple of rules for several selectors. Instead, you should create only one class

File: style.css [hunk 4]
```diff
@@ -0,0 +1,1043 @@
...
+    width: 90%;
+  }
+
+  .latest-news-grid img {
+    height: 11rem;
+  }
+
+  .latest-deals-grid div img {
+    height: 11rem;
+  }
```
Comment (andresporras3423) [inline style.css]: These selectors have the same style definition. Group selectors to minimize code.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,1043 @@
...
+  .mid-article-head-title ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    height: 1rem;
+  }
+
+  .last-child {
+    display: none;
+  }
```
Comment (andresporras3423) [inline style.css]: This selector and selector in line 774 have the same style definition. Group selectors to minimize code.

File: style.css [hunk 6]
```diff
@@ -0,0 +1,1043 @@
...
+
+  .nav-2-ul-2 {
+    width: 100%;
+    height: 3rem;
+  }
+
+  .nav-2-ul-2 li:last-child {
+    display: none;
+  }
```
Comment (andresporras3423) [inline style.css]: Three selectors with the same style definition, group selectors to minimize code.

File: style.css [hunk 7]
```diff
@@ -0,0 +1,1043 @@
...
+
+  .main-article-head-container {
+    display: none;
+    height: 4rem;
+  }
+
+  .hero-article {
+    display: block;
+  }
```
Comment (andresporras3423) [inline style.css]: Same style definition of the selector on line 857. Group selectors to minimize code.

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @AlduLonghi @descholar-ceo** 
Good job with your new submission for this project :muscle:  
Your page really  looks like a clone of the original :clap: :clap: :clap:  
Your project passes most of the requirements. There are just some details to improve in your CSS file before you can go ahead:
- [x] Don't repeat yourself, in order to minimize code you can group selectors with the same style definition (in your CSS files).
- [x] Line 607 of your CSS file, you are using the same couple of rules for several selectors. Instead, you should create only one class.

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

File: style.css (hunk 1 above)
Comment (descholar-ceo) [reply style.css]: Implemented this

File: style.css (hunk 2 above)
Comment (descholar-ceo) [reply style.css]: Implemented

File: style.css (hunk 3 above)
Comment (descholar-ceo) [reply style.css]: Implemented this

File: style.css (hunk 4 above)
Comment (descholar-ceo) [reply style.css]: Implemented

File: style.css (hunk 5 above)
Comment (descholar-ceo) [reply style.css]: Implemented

File: style.css (hunk 6 above)
Comment (descholar-ceo) [reply style.css]: Implemented

File: style.css (hunk 7 above)
Comment (descholar-ceo) [reply style.css]: Implemented

**Review by HADDADSOHAIB — APPROVED** [review]
### Status: Project Approved.

### Hi @AlduLonghi  and @descholar-ceo  :hand: 

### You did great with the project, you can merge this branch into master.

**Pull request merged**